src/platform/packages/shared/kbn-dom-drag-drop/src/sass/droppable.scss

```diff
@@ -51,14 +51,14 @@
 }
 
 .domDroppable--incompatible:not(.domDroppable__overlayWrapper) {
-  background-color: $euiColorHighlight !important;
+  background-color:  transparentize($euiColorWarning, .9) !important;
   &:before {
-    border: $euiBorderWidthThin dashed $euiColorVis5 !important;
+    border: $euiBorderWidthThin dashed $euiColorWarning !important;
   }
   &.domDroppable--hover {
-    background-color: rgba(251, 208, 17, .25) !important;
+    background-color: transparentize($euiColorWarning, .75) !important;
```

Comment:
Rather than applying transparency to `$euiColorWarning`, would it be better to use `$euiColorBackgroundLightWarning` here?

Reply:
done in [49d8096](https://github.com/elastic/kibana/pull/204839/commits/49d8096ab969b7f4b1a834b4ebba60b373c80808)